examples/js/loaders/GLTFLoader.js

```diff
@@ -2441,12 +2441,18 @@ THREE.GLTFLoader = ( function () {
 
 			var threeAttributeName = ATTRIBUTES[ gltfAttributeName ];
 
-			if ( ! threeAttributeName ) continue;
```

Comment:
I think this could be simplified a bit, to...

```js
var threeAttributeName = ATTRIBUTES[ gltfAttributeName ] || gltfAttributeName.toLowerCase();
```

... this lets us skip the extra if/else below.